### DISCUSSION TITLE
Homebrew won't install Mac M1 | Failed during: /usr/bin/sudo ...

### DISCUSSION THREAD

**Discussion by Rawkins711** [Homebrew/discussions#3751] [Getting started]
I am getting this error after running the install script, entering my password and clicking enter to continue creating the directories they list;

==> /usr/bin/sudo /usr/bin/install -d -o root -g wheel -m 0755 /opt/homebrew
install: mkdir /opt/homebrew: No such file or directory
Failed during: /usr/bin/sudo /usr/bin/install -d -o root -g wheel -m 0755 /opt/homebrew

Any help would be much appreciated.

**Comment by SMillerDev** [comment]
What does `la -al /opt` say?

**Reply by Rawkins711** [reply]
I installed an update for xcode and restarted my computer and it is good to go now. Thank you for your response though!
